src/common/user-store/preferences-helpers.ts

```diff
@@ -36,17 +36,18 @@ export interface KubeconfigSyncValue {
 }
 
 export type EditorConfiguration = Pick<editor.IStandaloneEditorConstructionOptions,
-  "minimap" | "tabSize" | "lineNumbers">;
+  "minimap" | "tabSize" | "lineNumbers" | "fontSize" | "fontFamily">;
 
 export const defaultEditorConfig: EditorConfiguration = {
   tabSize: 2,
   lineNumbers: "on",
+  fontSize: 12,
+  fontFamily: defaultFontFamily,
```

Comment:
should this var be called `defaultTerminalFontFamily` instead?

Also, given that it really should only be used here, I would move those vars into this file and not export them